src/utils/api/Api.tsx

```diff
@@ -5,8 +5,16 @@ const makeApiRequestToRKIData = async (url: string) => {
   return data;
 };
 
+const makeIncidentRequest = async (url: string) => {
+  const response = await (await fetch(url)).json();
+  const data: number = response.features[0].attributes.cases7_per_100k.toFixed(1); // this should work
```

Comment:
funktioniert hier die "normale" funktion mit `response.features[0].attirbutes.value` nicht? komisch, ist aber egal.

was ich nicht so schön finde: die frage ist, ob `toFixed(1)` hier sinn macht. sobald ich mit den zahlen rechnen will, habe ich ein problem, weil die funktion immer ein string zurück gibt. `toFixed(1)` einfach auslagern? meinetwegen in einer eigenen funktion in utils oder so

Reply:
Zu deiner Frage: Nein die "normale" funktion funktioniert nicht.
Bei dieser request URL findet sich unter `attributes` nicht nur `value`.